mmv1/products/cloudidentity/Group.yaml

```diff
@@ -142,12 +153,104 @@ properties:
     name: 'labels'
     required: true
     description: |
-      One or more label entries that apply to the Group. Currently supported labels contain a key with an empty value.
+      Required. One or more label entries that apply to the Group.
+      [Currently supported labels] contain a key with an empty value.
 
-      Google Groups are the default type of group and have a label with a key of cloudidentity.googleapis.com/groups.discussion_forum and an empty value.
+      * Google Groups are the default type of group and have a label with a key of
+      `"cloudidentity.googleapis.com/groups.discussion_forum"` and an empty value.
 
-      Existing Google Groups can have an additional label with a key of cloudidentity.googleapis.com/groups.security and an empty value added to them. This is an immutable change and the security label cannot be removed once added.
+      * Existing Google Groups can have an additional label with a key of
+      `"cloudidentity.googleapis.com/groups.security"` and an empty value added to
+      them. Adding the Security label to a group makes it a [security group]. This
+      action is permanent and adds security features, but doesn’t remove any other
+      features of the original group.
 
-      Dynamic groups have a label with a key of cloudidentity.googleapis.com/groups.dynamic.
+      * [Dynamic groups] have a label with a key of
+      `"cloudidentity.googleapis.com/groups.dynamic"` automatically added by the API.
+
+      * [Identity-mapped groups] for Cloud Search have a label with a key of
+      `"system/groups/external"` and an empty value.
+
+      **Note**: The Identity API may add additional labels that were not provided in
+        your config. If terraform plan shows a diff where a label is added, you can
+        add it to your config or apply the `lifecycle.ignore_changes` rule to the
+        labels field.
+
+      [Currently supported labels]:
+        https://cloud.google.com/identity/docs/reference/rest/v1/groups#Group.FIELDS.labels
+      [security group]:
+        https://support.google.com/a/answer/10607394
+      [Dynamic groups]:
+        https://cloud.google.com/identity/docs/concepts/overview-dynamic-groups
+      [Identity-mapped groups]:
+        https://cloud.google.com/identity/docs/groups#group-types
+  - !ruby/object:Api::Type::NestedObject
+    name: 'dynamicGroupMetadata'
+    immutable: true
```

Comment:
Can we propagate `immutable` down to the field's children or add an update test? Setting it at the top layer doesn't imply the children are immutable, as it's sometimes valid to update a child value but not a parent value.

Reply:
Ah, actually, I think I might need to remove this line? It looks like the API supports PATCH operations to update this.
https://cloud.google.com/identity/docs/how-to/create-dynamic-groups#python_1

Since that appears to be the case, I'll write some tests to confirm.

Reply:
Sounds good!